cxx-sensors/src/main/java/org/sonar/cxx/sensors/coverage/BullseyeParser.java

```diff
@@ -200,46 +194,86 @@ private void saveConditions(CoverageMeasures fileMeasuresBuilderIn) {
     totalcoveredconditions = 0;
   }
 
-  private void updateMeasures(String kind, String event, String line, CoverageMeasures fileMeasuresBuilderIn) {
+  private static void updateMeasures(String kind, String event, String line, CoverageMeasures fileMeasuresBuilderIn) {
 
-    if ("decision".equalsIgnoreCase(kind) || "condition".equalsIgnoreCase(kind)) {
-      if ("condition".equalsIgnoreCase(kind)) {
+    switch (kind.toLowerCase(Locale.ENGLISH)) {
```

Comment:
@guwirth The toLowerCase function is only called once for every single entry of kind or event. Did I oversee one?